pkg/maglev/maglev.go

```diff
@@ -63,15 +66,7 @@ func getOffsetAndSkip(backend string, m uint64) (uint64, uint64) {
 }
 
 func getPermutation(backends []string, m uint64, numCPU int) []uint64 {
-	var wg sync.WaitGroup
-
-	// The idea is to split the calculation into batches so that they can be
```

Comment:
Please keep the comment since this is not changing how the calculation works. Don't link to the issue.